backend/snk_web/routes/api/index.js

```diff
@@ -6,6 +6,8 @@ const clienteRoutes = require("./cliente");
 const produtoRoutes = require("./produto");
 const carrinhoRoutes = require("./carrinho");
 const enderecoRoutes = require("./endereco");
+const pedidoRoutes = require("./pedido");
+const ItemCarrinhoRoutes = require("./item_carrinho");
```

Comment:
Nome que começam com letra maiúscula são voltadas pra Classes, aqui podemos manter como minúsculo mesmo: `itemCarrinhoRoutes`

```suggestion
const itemCarrinhoRoutes = require("./item_carrinho");
```